src/generic/FunctionField.jl

```diff
@@ -528,7 +528,8 @@ parent_type(::Type{FunctionFieldElem{T}}) where T <: FieldElement = FunctionFiel
 elem_type(::Type{FunctionField{T}}) where T <: FieldElement = FunctionFieldElem{T}
 
 function base_ring(R::FunctionField{T}) where T <: FieldElement
-   return R.base_ring::RationalFunctionField{T}
+   U = dense_poly_type(T)
+   return R.base_ring::RationalFunctionField{T, U}
```

Comment:
That means that a function field field is always a finite extension of a univariate rational function field, which is ok. But then all the function field methods below should not allow `MPolyElem`.

Reply:
I guess so. I've excluded them for now.